webpack/module-minifier-plugin/src/ModuleMinifierPlugin.ts

```diff
@@ -509,9 +493,31 @@ export class ModuleMinifierPlugin implements webpack.Plugin {
           }
         );
 
-        for (const template of [compilation.chunkTemplate, compilation.mainTemplate]) {
-          (template as unknown as IExtendedChunkTemplate).hooks.modules.tap(TAP_AFTER, dehydrateAsset);
-        }
+        (compilation.chunkTemplate as unknown as IExtendedChunkTemplate).hooks.modules.tap(
```

Comment:
Will these two ever be tapped differently from eachother? Would it make sense to create a common func to share the logic?

Reply:
This was done because the signatures don't match. The `moduleTemplate` parameter is 3rd in `chunkTemplate`, 4th in `mainTemplate`.

Reply:
Yeah, that makes sense for being why you changed it. I'm talking about the contents of the tap. You could still call out into the same func for both though.